Readme.md

```diff
@@ -93,6 +99,31 @@ Parallel.each(User.all, in_processes: 8) do |user|
 end
 ```
 
+You may also want to bump up your connection pool size in
+`config/database.yml`.
+
+#### Unexpected and Random "NameError: unintialized constant"
+
+There's a potential race when ActiveRecord models are going to be autoloaded
+from within Parallel code blocks. This is mostly only visible in environments
+non preloading the whole application, i.e. while migrating.
```

Comment:
```suggestion
non preloading the whole application, like development/test/migrations.
```
... FYI migrations should usually not use existing models since that will make them break when someone runs old migrations

Reply:
> ... FYI migrations should usually not use existing models since that will make them break when someone runs old migrations

Yes, that's currently one problem I'm facing. But it's actually needed because I have to migrate from a legacy database scheme to a modern one to clean up a lot of crufty code. My solution is to leave the models as compatible as I can get and put todos notes in the files to remove some stuff after deployment. Later on after deployment, we can get rid of those migrations and solely rely on `schema.rb` to rebuild an empty database.

Because migrations are huge (and it takes my dev machine around 4-5 hours to run them), I've integrated parallel to get it down to around 1 hour. I regularly need to reset the database by fetching it again from production.

Thanks for the pointer, tho. :-)